### PR TITLE
Basic tests for UsersTab add user button

### PR DESCRIPTION
**Pull request by aviav**

### CODE REVIEW

File: spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module Pageflow
+  module Admin
+    describe AddMembershipButtonIfNeeded, type: :view_component do
+      before do
+        helper.extend(ActiveAdmin::ViewHelpers)
+        helper.extend(Pageflow::Admin::MembershipsHelper)
+        allow(helper).to receive(:new_admin_account_membership_path)
+        allow(helper).to receive(:membership_possible_for)
```
Comment (tf) [inline spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb]: `membership_possible_for` is a private method of the view component not a helper method. Does this `allow` do anything?

File: spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module Pageflow
+  module Admin
+    describe AddMembershipButtonIfNeeded, type: :view_component do
+      before do
+        helper.extend(ActiveAdmin::ViewHelpers)
+        helper.extend(Pageflow::Admin::MembershipsHelper)
+        allow(helper).to receive(:new_admin_account_membership_path)
```
Comment (tf) [inline spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb]: Isn't stubbing `url_for` enough?

File: spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb (hunk 1 above)
Comment (aviav) [inline spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb]: Here as well as with `url_for`, the exception backtrace changed. The individual specs did already pass without these calls, and maybe running the test suite again I will find that having `url_for` stubbed makes the suite pass already

File: spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb (hunk 2 above)
Comment (aviav) [inline spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb]: Tested by running `bin/rspec`. This line is needed

File: spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb (hunk 1 above)
Comment (aviav) [inline spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb]: Tested by running `bin/rspec`. This line is indeed *not* needed

**Pull request merged**